src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -12,7 +12,7 @@
   "precedence": "9000",
   "identity": "Microsoft.Web.Blazor.Server.CSharp.7.0",
```

Comment:
Does this need to be bumped to 8.0 as well?

Reply:
Yes, I'm planning on doing name/precedence updates in a separate PR

Reply:
I'm not sure the packages are already available or not, so for now will not touch these.